Src/BootCamp.Chapter/Inventory.cs

```diff
@@ -1,35 +1,54 @@
-﻿namespace BootCamp.Chapter
+﻿using System.Collections.Generic;
+
+namespace BootCamp.Chapter
 {
     public class Inventory
     {
-        private Item[] _items;
-        public Item[] GetItems()
-        {
-            return new Item[0];
-        }
+        public List<Item> Items { get; private set; }
```

Comment:
private set can be removed. If a property is get only it can still be set, but only in ctor. This is exactly what you want.